app/src/main/java/com/infomaniak/mail/data/LocalSettings.kt

```diff
@@ -18,21 +18,60 @@
 package com.infomaniak.mail.data
 
 import android.content.Context
-import androidx.annotation.ColorRes
-import androidx.annotation.StringRes
-import androidx.annotation.StyleRes
+import androidx.annotation.*
 import androidx.appcompat.app.AppCompatDelegate
 import com.infomaniak.lib.core.utils.Utils.enumValueOfOrNull
 import com.infomaniak.lib.core.utils.transaction
 import com.infomaniak.mail.R
 import kotlin.reflect.KMutableProperty0
 
-class UiSettings private constructor(context: Context) {
+class LocalSettings private constructor(context: Context) {
```

Comment:
Use hardcoded keys everywhere (for the SharedPref file itself, and for each value).